servers/src/main/java/tachyon/master/next/block/meta/MasterBlockLocation.java

```diff
@@ -18,12 +18,20 @@
 /**
  * The location of the block.
  */
-public class MasterBlockLocation {
-  public long mWorkerId;
-  public int mTier;
+public final class MasterBlockLocation {
+  private final long mWorkerId;
+  private final int mTier;
```

Comment:
That is a good question. When a block is moved from one tier to another, how does that get reflected in the worker heartbeat?